proxyd/backend.go

```diff
@@ -746,8 +755,13 @@ func (b *Backend) doForward(ctx context.Context, rpcReqs []*RPCReq, isBatch bool
 	start := time.Now()
 	httpRes, err := b.client.DoLimited(httpReq)
 	if err != nil {
-		b.intermittentErrorsSlidingWindow.Incr()
-		RecordBackendNetworkErrorRateSlidingWindow(b, b.ErrorRate())
+		if !(errors.Is(err, context.Canceled) || errors.Is(err, ErrTooManyRequests)) {
```

Comment:
Should the second half of the conditional also contain a negation?
```suggestion
		if !(errors.Is(err, context.Canceled) || !errors.Is(err, ErrTooManyRequests)) {
```

Reply:
I think the current logic is correct

if it's untrue that it's either context canceled or too many requests, then it's a real error and we should track it

the negation is for both errors already